projects/04_cube_xr/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
+
+project(04_cube_xr)
+
+add_samples_for_all_apis(
```

Comment:
This has to be `add_samples`, not `add_samples_for_all_apis`, if you specify the APIs.

Reply:
fixed